.github/workflows/pipeline.yaml

```diff
@@ -0,0 +1,76 @@
+name: Build, test and push
+on:
+  push:
+    branches-ignore:
+      - "release/**"
+jobs:
+  build_image:
+    needs:
+      - check_and_test
+    strategy:
+      matrix:
+        image:
+          - name: synthetic-pipeline
+            dockerfile: Dockerfile.psga-pipeline
+            pathogen: synthetic
+          - name: sars-cov-2-pipeline
+            dockerfile: Dockerfile.psga-pipeline
+            pathogen: sars_cov_2
+          - name: fastqc
+            pathogen: sars_cov_2
+          - name: read-it-and-keep
+            pathogen: sars_cov_2
+          - name: primer-autodetection
+            pathogen: sars_cov_2
+          - name: pangolin
+            pathogen: sars_cov_2
+          - name: ncov2019-artic-nf-nanopore
+            pathogen: sars_cov_2
+          - name: ncov2019-artic-nf-illumina
+            pathogen: sars_cov_2
+    uses: ./.github/workflows/build_image.yaml
+    secrets: inherit
+    with:
+      name: ${{ matrix.image.name }}
+      dockerfile: ${{ matrix.image.dockerfile }}
+      pathogen: ${{ matrix.image.pathogen }}
+  publish_configs_dev:
```

Comment:
As I believe Piero has pointed out - it would be good to push images for feature branches as well, as it allows us to test them out in dev before merging. Can we add that please?

Reply:
It now pushes images and configs from branches too.